Minor tweaks

tianon opened the pull request:
I've split this up into several commits to help make it easier to review and talk about each part:

- b54c2ed04c3ff4e3813e1bd88c4105b3e0cdea64: use `--recv-keys` on each key directly instead of trusting the remote `KEYS` file so that each fingerprint is independently verified to be exactly what we expected at `Dockerfile` creation time
- bb2e1336f0564b905a4e48a06257fc10d9a6758b: combine the `.asc` downloading `RUN` into the following `RUN` line which deletes the associated `.asc` file so that it gets removed properly and not just obscured via Docker's layer whiteout, add `-f` to all the `curl` invocations so that `curl` will exit non-zero if there's an issue with the download, and verify the tarball before we extract it
- 5b475887d8f1c926df41740471bc86819d5825a3: adjust `create-server.sh` so that it can also be used to update an existing `Dockerfile` so that changes to `Dockerfile.template` can be more easily distributed to all the individual `Dockerfile`s
- 986e80301ac11cfaa7bf5554ff2516badcda8d96: actually regenerates all the `Dockerfile`s from the template :+1:

yosifkit commented on Dockerfile.template:
```diff
@@ -5,13 +5,27 @@ RUN mkdir -p /usr/local/tomee
+	&& curl -fSL https://dist.apache.org/repos/dist/release/tomee/tomee-%VERSION%/apache-tomee-%VERSION%-%FLAVOR%.tar.gz.asc -o tomee.tar.gz.asc \
+	&& curl -fSL http://apache.rediris.es/tomee/tomee-%VERSION%/apache-tomee-%VERSION%-%FLAVOR%.tar.gz -o tomee.tar.gz \
+	&& tar -zxf tomee.tar.gz  \
 	&& gpg --verify tomee.tar.gz.asc tomee.tar.gz \
```
Extract and then verify? :wink:

tianon replied:
Doh, great catch.  Will update shortly.

tianon commented:
Updated PR for @yosifkit's comment and updated initial PR description to match:
> - bb2e1336f0564b905a4e48a06257fc10d9a6758b: combine the `.asc` downloading `RUN` into the following `RUN` line which deletes the associated `.asc` file so that it gets removed properly and not just obscured via Docker's layer whiteout, add `-f` to all the `curl` invocations so that `curl` will exit non-zero if there's an issue with the download, and verify the tarball before we extract it

lordofthejars commented:
Thank you so much, I really appreciate this help.
